src/android/nl/xservices/plugins/Calendar.java

```diff
@@ -513,7 +495,7 @@ public void run() {
           calendar_end.setTime(date_end);
 
           //projection of DB columns
-          String[] l_projection = new String[]{"calendar_id", "title", "begin", "end", "eventLocation", "allDay", "_id"};
+          String[] l_projection = new String[]{"calendar_id", "title", "begin", "end", "eventLocation", "allDay"};
```

Comment:
I am not sure, I didn't change this area !!

I will check tomorrow
On May 23, 2016 10:43 PM, "Eddy Verbruggen" <notifications@github.com>
wrote:

> In src/android/nl/xservices/plugins/Calendar.java
> <https://github.com/EddyVerbruggen/Calendar-PhoneGap-Plugin/pull/293#discussion_r64275947>
> :
>
> > @@ -513,7 +495,7 @@ public void run() {
> >            calendar_end.setTime(date_end);
> >
> >            //projection of DB columns
> > -          String[] l_projection = new String[]{"calendar_id", "title", "begin", "end", "eventLocation", "allDay", "_id"};
> > +          String[] l_projection = new String[]{"calendar_id", "title", "begin", "end", "eventLocation", "allDay"};
>
> Where did "_id" go?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/EddyVerbruggen/Calendar-PhoneGap-Plugin/pull/293/files/4660ce3b561ddb2231a2a84a8126fba2c74ba30b#r64275947>
>